FeedStoreChallenge/GRDBFeedStore/GRDBFeedStore.swift

```
@@ -0,0 +1,85 @@
+
+//  GRDBFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by 游宗諭 on 2020/11/27.
+//  Copyright © 2020 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import GRDB
+
+
+
+public class GRDBFeedStore: FeedStore {
+    
+    internal let dbQueue: DatabaseQueue
+    private var dbManeger: DBQueueManager
+    
+    public init(path: String?, dbManeger: DBQueueManager) throws {
+        self.dbManeger = dbManeger
+        dbQueue = try dbManeger.makeQueue(path: path)
+    }
+    deinit {
+        dbManeger.recylcle(queue: dbQueue)
+    }
+    public func deleteCachedFeed(completion: @escaping DeletionCompletion) {
+        try? dbQueue.write({ (db) in
```

Errors should be handled. But `try?` will ignore errors.

---

Sure, the issue has been fixed by follow up commit.